docs/semgrep-assistant/customize.md

```diff
@@ -0,0 +1,205 @@
+---
+slug: customize
+title: Customize Assistant
+hide_title: true
+description: Learn how to enable and configure Assistant's features.
+tags:
+  - Deployment
+  - Semgrep Assistant
+---
+
+# Customize Semgrep Assistant
+
+You can customize Semgrep Assistant by enabling and using the features detailed on this page.
+
+## Remediation
+
+Assistant remediation allows you to receive AI-generated code snippets for true positives. Perform the following to enable it:
+
+1. Sign in to Semgrep AppSec Platform, and navigate to **Settings > Deployment**.
+2. In the **Assistant** section, click the **Remediation** <i class="fa-solid fa-toggle-large-on"></i> if it is not yet enabled.
+3. *Optional*: Select a **confidence level** in the drop-down box. This value determines the level of quality at which the autofix code appears as a suggestion. A lower confidence level means that Semgrep Assistant displays the autofix suggestion even when the code quality may be incorrect.
+
+:::tip
+Semgrep recommends setting a low confidence level since even incorrect suggestions may be useful starting points for triage and remediation.
+:::
+
+## Auto-triage
+
+If [auto-triage](/semgrep-assistant/overview/#auto-triage), which allows you to get notifications whenever Assistant indicates a finding may be safe to ignore, isn't enabled for your deployment, you can do so as follows:
```

Comment:
Is auto-triage on by default?

Reply:
Yeah, by default, all features are enabled after enabling Assistant. I think the only way this would be disabled is if someone did so manually. 
![image](https://github.com/user-attachments/assets/34900b5f-8374-4e33-9af4-23c5ca08d4ef)

Reply:
Maybe the way to handle this is to reorder the instructions:

> [Auto-triage](/semgrep-assistant/overview/#auto-triage) allows you to get notifications whenever Assistant indicates a finding may be safe to ignore. If this feature isn't enabled for your deployment, you can do so using the following instructions

WDYT?

Reply:
SGTM!